patches/net/minecraft/client/renderer/PostChain.java.patch

```diff
@@ -0,0 +1,38 @@
+--- a/net/minecraft/client/renderer/PostChain.java
++++ b/net/minecraft/client/renderer/PostChain.java
+@@ -95,17 +_,33 @@
+         Matrix4f matrix4f = new Matrix4f().setOrtho(0.0F, (float)p_361423_, 0.0F, (float)p_362735_, 0.1F, 1000.0F);
+         Map<ResourceLocation, ResourceHandle<RenderTarget>> map = new HashMap<>(this.internalTargets.size() + this.externalTargets.size());
+ 
++        // Enable the depth and stencil buffers based on whether any external targets use them.
```

Comment:
Not sure what this patch is trying to achieve.

Reply:
Exactly what the comment says - it enables the depth/stencil buffers of created resources in the post chain based on whether the inputs use them. This is to ensure blitting and framebuffer copying works correctly.

Reply:
Further simplified to match how vanilla just passes `true` for depth.